### PR TITLE
Add weekly update 2016-11-11

### PR DESCRIPTION
**Pull request by JungMinu**
Please review this post. :heart:,@nodejs/evangelism

### CODE REVIEW

**Comment by vdeturckheim** [comment]
LGTM :smile:

File: locale/en/blog/weekly-updates/weekly-update.2016-11-11.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+* [6 of the Most Exciting ES6 Features in Node.js v6 LTS](https://nodesource.com/blog/six-of-the-most-exciting-es6-features-in-node-js-v6-lts): With this article, we'll take a look at six of the best new ES6 features that are in the Node.js v6 LTS release.
+
+If you have spotted or written something about Node.js, do come over to our [Evangelism team repo](https://github.com/nodejs/evangelism) and suggest it on the [Issues page](https://github.com/nodejs/evangelism/issues), specifically the Weekly Updates issue.
+
+### Upcoming Events
+
+* [NodeConf Barcelona 2016](http://barcelona.nodeconf.com/): 18th November 2016 · Barcelona Mobile World Centre
+* [NodeConf Argentina](https://2016.nodeconf.com.ar): 18 - 19 November, 2016
+* [play.node 2016](http://playnode.io/2016/): 23rd November 2016 · Seoul, South Korea
+* [EventLoop](http://eventloop.noders.com/): 26th - 27th November, 2016
```
Comment (Trott) [inline locale/en/blog/weekly-updates/weekly-update.2016-11-11.md]: This event needs a location. I believe it is in Santiago, Chile.

**Comment by Knighton910** [comment]
LGTM :)

**Review by mhdawson — APPROVED** [review]
LGTM for the bechmarking and code coverage secctions

**Comment by JungMinu** [comment]
@Trott Done and updated :)

**Pull request merged**